meson.build

```diff
@@ -137,7 +137,23 @@ else
     ]
 endif
 
-
+# Embed binary files into the program
+# converting them to source as `const array` definitions
+
+embedder = find_program('embed/embedder.py', required: false)
+if get_option('embed') and embedder.found()
+  # Rule to convert binary files to source + header.
+  embedder_gen = generator(embedder,
+    output : ['@BASENAME@.cpp', '@BASENAME@.h'],
+    arguments : ['@INPUT@', '@OUTPUT0@', '@OUTPUT1@']
+  )
+  # Add network weights file if found
+  weights_file = 'embed/embedded_weights'
+  if run_command('test', '-f', weights_file).returncode() == 0
```

Comment:
This doesn't look like it will work on windows. [checkdir.py](https://github.com/LeelaChessZero/lc0/blob/master/checkdir.py) was implemented to check the existence of directories in a corss-platform way, so a similar approach should work here as well.